app/scripts/directives/configEditor.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .directive('configEditor', function(APIService, DataService) {
```

Comment:
`mobileConfigEditor` as the name?

I would change to `.component` now as we don't use `link`

Reply:
if I change this to `.component` it stops working